integration-test/aot.Tests.ps1

```diff
@@ -50,11 +50,22 @@ Console.WriteLine("Hello, Sentry!");
     }
 
     It 'Aot' {
-        dotnet publish -c Release | Write-Host
+        $rid = $env:RuntimeIdentifier
+        if ($env:RuntimeIdentifier)
```

Comment:
nitpick: check `$rid` instead of `$env:RuntimeIdentifier`

I am wondering if it's more clear to use `$rid` for the check,
since below, we also use `if (-not $rid)`.

But I guess that's very much a style question.
My care-factor is neglectable 😉